app/src/test/java/by/academy/jc/ht/klimakhovich/MyStackTest.java

```diff
@@ -0,0 +1,50 @@
+package by.academy.jc.ht.klimakhovich;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class MyStackTest {
```

Comment:
Requirement

> Implement java.util.List interface according to LIFO principle 

was not covered with unit tests. 

YOu had to use stack in for-each construct, for example